controllers/suite_test.go

```diff
@@ -111,19 +112,31 @@ func afterSuite() error {
 // duplicateRequest returns a reconcile.Reconcile implementation that delegates to inner and
 // writes the request to requests after Reconcile is finished.
 func duplicateRequest(t *testing.T, inner reconcile.Reconciler, stopped *bool) (reconcile.Reconciler, chan reconcile.Request, chan reconcile.Result, chan error) {
-	requests := make(chan reconcile.Request)
-	results := make(chan reconcile.Result)
-	errors := make(chan error)
+	requests := make(chan reconcile.Request, 100)
```

Comment:
why do we need buffered channels here? why with a capacity of 100?

Reply:
I think you are right and we don't need buffered channels since we handle the case of the channels not beeing consumed already...